index.html

```diff
@@ -60,28 +75,41 @@ <h2>My Recent Works</h2>
 
           <div class="wrapper">
             <div class="feature">
-              <img src="src/Img-Placeholder-n1.svg" alt="Feature stories" />
-              <h3>Multi-Post Stories</h3>
-              <p class="feat-text">
-                A daily selection of privately personalized reads; no accounts
-                or sign-ups required. has been the industry's standard dummy
-                text ever since the 1500s, when an unknown printer took a
-                standard dummy text.
-              </p>
-              <ul class="tech-stack-1">
-                <li>CSS</li>
-                <li>HTML</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
-              </ul>
-              <button class="btn">See Project</button>
+              <div class="feature-img">
+                <img
+                  src="src/Img-Placeholder-n1.svg"
+                  alt="Feature stories"
+                  class="mob-img"
+                />
```

Comment:
- [ ] I would suggest updating the size for the "featured project" image because it's showing too small in your portfolio and I think that it could take around 60% considering the template :pushpin: :smiley: 

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144723130-6bc4acec-4ede-4f26-84e8-287edca7e84e.png)

  Figma design:
  ![image](https://user-images.githubusercontent.com/81522435/144723135-574030c8-4a6a-4e01-ab41-28a4867b5b09.png)